CHANGES.md

```diff
@@ -3,6 +3,11 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.399
+- Remove `StatusBoard` `fullname` search parameter.
+- Deprecate `Notify` resource.  
+- Update model version v0.0.352.
```

Comment:
Please see example on how to document this: https://github.com/openshift-online/ocm-sdk-go/blob/main/CHANGES.md#01391